### ISSUE TITLE
Make Programs taggable with keywords and add interface search/filter by keyword in database

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/papyrus#36]
While we have plans to organize sets of Program examples by Playspace, we'd like to be able to filter the Programs by categories.

Currently, the Programs are named based on a comment written on Line 1. It was suggested that these keywords be added as the second comment line (Line 2) of the Program. Possibly comma separated or whatever is best for reading it back out with an SQL query.

#### Database

- [x] Add the keyword tagging ability and have it updated in the database (a new column?)
- [x] Update our current examples

#### Interface 

- [x] Spend the time to create a simple interface search/filter to only display Programs that contain the desired keywords

Linking this to #25 to determine where this fits into the interface as a whole. We can probably put this into the Editor page for now. 

We'll also (later on) need to decide how this behaves with #34, which may let us reach across Playspaces without leaving the current one. Depending on how that is implemented, we might want to filter whatever view we have into the "remote" Playspace. However, this is not slated currently for implementation.

**Comment by BLFiedler** [comment]
During a meeting today (BF, JB, JG), we discussed prioritizing this feature next.

We'll want the search capability in Camera and the Editor (both places you must navigate a list of Programs).

JB is going to remove the print queue in Camera, push the core Printing functions down on the sidebar, and integrate the keyword search with the list of programs used for Previewing.

I made a note of the following regarding how we create the keywords:
-  Let's start with user-generated (need to watch for typos). When we have generated enough keywords, we can group and theme as we see appropriate. Perhaps then we provide a predefined list that is recognized. Difficult to anticipate the breadth of needed categories right now.
